deploy/07_erc721drop.ts

```diff
@@ -88,7 +188,7 @@ const func: DeployFunction = async function (hre1: HardhatRuntimeEnvironment) {
       ['tuple(address,address,address,string,string,address,address,uint64,uint16,bytes[],address,bytes)', 'bool'],
       [
         [
-          futureFeeManagerAddress, // holographFeeManager
+          '0x0000000000000000000000000000000000000000', // holographFeeManager
```

Comment:
I thought fee manager was removed from the new DropsInitializer? Or is this for the old drops contract

Reply:
This is for old_drops deployment code. There’s a TODO comment above that says this should all be deleted once we remove old_drops.